vars/mtaBuild.groovy

```diff
@@ -38,26 +38,26 @@ def call(Map parameters = [:]) {
         def mta = new JavaArchiveDescriptor('SAP Multitarget Application Archive Builder', 'MTA_JAR_LOCATION', 'mtaJarLocation', '1.0.6', '-v', java)
         mta.verify(this, configuration)
 
-        def mtaYmlName = "${pwd()}/mta.yaml"
+        def mtaYmlName = "mta.yaml"
         def applicationName = configuration.applicationName
 
         if (!fileExists(mtaYmlName)) {
             if (!applicationName) {
                 echo "'applicationName' not provided as parameter - will not try to generate mta.yml file"
             } else {
                 MtaUtils mtaUtils = new MtaUtils(this)
-                mtaUtils.generateMtaDescriptorFromPackageJson("${pwd()}/package.json", mtaYmlName, applicationName)
+                mtaUtils.generateMtaDescriptorFromPackageJson("package.json", mtaYmlName, applicationName)
             }
         }
 
-        def mtaYaml = readYaml file: "${pwd()}/mta.yaml"
+        def mtaYaml = readYaml file: "mta.yaml"
```

Comment:
We should use the variable mtaYmlName again instead of defining the same string again, or is there is reason not to use 
`def mtaYaml = readYaml file: mtaYmlName`

Same applies of the rest of the step.

Reply:
Yes, should be re-used. Was not like this before, hence it is not like this afterwards. But yes, should - of course -be changed. --> I will do so.

Reply:
done